Gems/ScriptCanvasTesting/Code/Tests/ScriptCanvas_FileHandling.cpp

```diff
@@ -10,9 +10,12 @@
 #include <Source/Framework/ScriptCanvasTestNodes.h>
 #include <Source/Framework/ScriptCanvasTestUtilities.h>
 #include <ScriptCanvas/Components/EditorGraph.h>
+#include <ScriptCanvas/Core/GraphSerialization.h>
 
 namespace ScriptCanvasTests
 {
+    using namespace ScriptCanvas;
+
```

Comment:
Repeat: the `using namespace ScriptCanvas;` call needs to be moved to function scope or removed altogether